consensus/core/src/core.rs

```diff
@@ -212,6 +226,13 @@ impl Core {
     /// or earlier round, then no block is created and None is returned.
     fn try_new_block(&mut self, force: bool) -> Option<VerifiedBlock> {
         let _scope = monitored_scope("Core::try_new_block");
+        let _s = self
+            .context
+            .metrics
+            .node_metrics
+            .scope_processing_time
+            .with_label_values(&["Core::try_new_block"])
+            .start_timer();
```

Comment:
Is there a private testnet run with these metrics? It seems the timer unit is millisec, which seems too coarse for Core operations.

Reply:
Yes there is , you can see an example below. However, as far as I know the reporting is not done in miliseconds , but rather in seconds + nanos. So the smallest possible metric should be nanos. You make a good point though which made me refine the buckets so we don't use the default ones. You can see on the screenshots below some metrics which manage to measure code scope in `μs` and `ms`

<img width="832" alt="Screenshot 2024-03-19 at 15 25 31" src="https://github.com/MystenLabs/sui/assets/17335598/e9cb0e49-d515-49a2-a35b-e55dc5142144">

<img width="830" alt="Screenshot 2024-03-19 at 15 25 36" src="https://github.com/MystenLabs/sui/assets/17335598/1abda519-cbbd-4a6f-9282-d219c8cddfa1">

Reply:
Good this is using prometheus HistogramVec instead of the internal one. I guess sum and count are indeed accurate and with the updated buckets, quantiles should be more accurate now.